.travis.yml

```diff
@@ -4,3 +4,15 @@ go: "1.11"
 install: true
 script:
 - GINKGO_EXTRA_ARGS="--compilers=2" make all test/unit
+
+deploy:
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make handler push-handler
+  skip_cleanup: true
+  on:
+    branch: master
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make handler push-handler HANDLER_IMAGE_SUFFIX=$TRAVIS_TAG
```

Comment:
We also need to make a github release, for that we were calling "make release" it was doing all, tagging, pushing and releasesing on github.
We can add "make github-release" it will just call https://github.com/nmstate/kubernetes-nmstate/blob/master/hack/release.sh without pushing or tagging and calling here.

Also the "script" sectoin is getting log better to using and script file or yaml multiline.

Reply:
this provider section runs after there is a new tag in github.

Reply:
What I have in mind is having a release "PR" with version and release description and when we merge that PR a script will tag/release/push for us, that is what "make release" do

Reply:
@qinqon with the `make github-release` script. Do we need to tag ourselves or is it done automatically? If automatically, we need to call it from the master branch.

Anyway, I think we could do it in a separate PR to keep things going and focused. Won't fight for it though :)